sdk/purview/azure-purview-datamap/MigrationGuide.md

```diff
@@ -0,0 +1,58 @@
+# Migrate from Purview Catalog to Purview DataMap
+
+This guide is intended to assist in the migration to Azure Purview DataMap client library [`azure-purview-datamap`](https://pypi.org/project/azure-purview-datamap/) from [`azure-purview-catalog`](https://pypi.org/project/azure-purview-catalog/). It will focus on side-by-side comparisons for similar operations between the two packages.
+
+For those new to the Purview Data Map library, please refer to the [`azure-purview-datamap` readme](https://github.com/Azure/azure-sdk-for-python/blob/293d480d3ce7e9fe233976a8c5c6c565c5c68665/sdk/purview/azure-purview-datamap/README.md) and [`azure-purview-datamap` samples](https://github.com/Azure/azure-sdk-for-python/blob/293d480d3ce7e9fe233976a8c5c6c565c5c68665/sdk/purview/azure-purview-datamap/samples) for the `azure-purview-datamap` library rather than this guide.
```

Comment:
please change the commit id to `main` branch name before merge.

Reply:
Having the failure as:
[404] broken link https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/purview/azure-purview-datamap/README.md (resolved to https://github.com/Azure/azure-sdk-for-python/blob/c664b3089b9839393991b6700a5752cbd40832c8/sdk/purview/azure-purview-datamap/README.md)